### PR TITLE
Re-enabling and fixing repairer tests

### PR DESCRIPTION
**Pull request by ethanadams**
What: 
Re-enabling and fixing repairer tests that would intermittently fail
Why:
To have proper test coverage for repair services

Thanks for submitting a PR!

## Code Review Checklist (to be filled out by reviewer)
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Could the PR be broken into smaller PRs?
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?

### CODE REVIEW

File: internal/testplanet/planet.go [hunk 1]
```diff
@@ -479,7 +479,7 @@ func (planet *Planet) newSatellites(count int) ([]*satellite.Peer, error) {
 			Repairer: repairer.Config{
 				MaxRepair:    10,
 				Interval:     time.Hour,
-				Timeout:      2 * time.Second,
+				Timeout:      10 * time.Second,
```
Comment (egonelbre) [inline internal/testplanet/planet.go]: Why was this change needed?

File: pkg/datarepair/datarepair_test.go [hunk 2]
```diff
@@ -28,10 +27,10 @@ func TestDataRepair(t *testing.T) {
...
 		ul := planet.Uplinks[0]
 		satellite := planet.Satellites[0]
-		// stop discovery service so that we do not get a race condition when we delete nodes from overlay cache
-		satellite.Discovery.Service.Discovery.Stop()
-		satellite.Discovery.Service.Refresh.Stop()
-		satellite.Discovery.Service.Graveyard.Stop()
+		// pause discovery service so that we do not get a race condition when we delete nodes from overlay cache
+		satellite.Discovery.Service.Discovery.Pause()
+		satellite.Discovery.Service.Refresh.Pause()
+		satellite.Discovery.Service.Graveyard.Pause()
```
Comment (egonelbre) [inline pkg/datarepair/datarepair_test.go]: It shouldn't make a difference whether you stop or pause.

Comment (ethanadams) [reply pkg/datarepair/datarepair_test.go]: I agree.  But I saw some issues when Stop was used.  Maybe I misread the issue.  I'll change back to Stop and see if I can reproduce again.

File: pkg/audit/newverifier_test.go [hunk 3]
```diff
@@ -283,6 +287,7 @@ func TestDownloadSharesDialTimeout(t *testing.T) {
 // If this test fails, this most probably means we made a backward-incompatible
 // change that affects the audit service.
 func TestDownloadSharesDownloadTimeout(t *testing.T) {
+	t.Skip()
```
Comment (egonelbre) [inline pkg/audit/newverifier_test.go]: should the audit tests be still skipped?

Comment (ethanadams) [reply pkg/audit/newverifier_test.go]: My mistake. Reverted "skip" code.
